packages/libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -84,6 +85,11 @@ const SOL_PER_LAMPORT = 0.000000001
 // Generous default connection confirmation timeout to better cope with RPC congestion
 const DEFAULT_CONNECTION_CONFIRMATION_TIMEOUT_MS = 180 * 1000
 
+const PRIORITY_RATE = 100000 // MICRO_LAMPORTS
```

Comment:
honestly, probably fine to just not declare this as a var and pass it to the microLamports below